server/base/src/main/java/org/apache/accumulo/server/ServerOpts.java

```diff
@@ -20,17 +20,11 @@
 
 import org.apache.accumulo.core.cli.ConfigOpts;
 
-import com.beust.jcommander.Parameter;
+public final class ServerOpts extends ConfigOpts {
 
-public class ServerOpts extends ConfigOpts {
+  // This class is empty on purpose. The intent here is that
+  // the Accumulo server processes will only ConfigOpts. Can't
+  // make ConfigOpts final as it's used by utility classes
+  // that subclass it.
```

Comment:
Can we just delete this class?

Reply:
Yes, but that would enable someone to reintroduce a subclass of ConfigOpts for a server process at a later time.

Reply:
If we need to re-add it in future, that's fine. If not, it stays deleted. It's internal code organization only, not public API, so it's okay to delete if we don't need it.

Reply:
removed in fae4135